Wrapping/Generators/Python/PyBase/pyBase.i

```diff
@@ -425,13 +425,18 @@ str = str
                 import numpy as np
                 import itkTypes
 
-                # numpy dtype
+                # if both a numpy dtype and a ctype exist, use the latter.
                 if type(pixel_type) is type:
-                    pixel_type = itkTypes.itkCType.GetCTypeForDType(pixel_type)
-                current_pixel_type = itk.template(self)[1][0]
-                if current_pixel_type is pixel_type:
+                    c_pixel_type = itkTypes.itkCType.GetCTypeForDType(pixel_type)
+                    if c_pixel_type is not None:
+                        pixel_type = c_pixel_type
+
+                # input_image_template is Image or VectorImage
```

Comment:
Please add a test for `itk.VectorImage`.

Reply:
Thank you.  I just added tests to `Wrapping/Generators/Python/Tests/extras.py` and pushed.